pkg/types/types.go

```diff
@@ -68,6 +68,11 @@ type BlockKey string
 
 type Address []byte
 
+type UpkeepObservation struct {
+	BlockKey          BlockKey           `json:"blockKey"`
+	UpkeepIdentifiers []UpkeepIdentifier `json:"upkeepIdentifiers"`
```

Comment:
Should we give smaller/more succinct JSON keys for these entries? 🤷‍♂️ It might be the difference of an extra upkeep being included?

Reply:
I would use "block" and "ids"

Reply:
yup. reduces p2p bandwidth too. I would be ok with 'b' and 'u' also.

Reply:
I think we're splitting hairs hairs here. We are limiting the number of upkeeps anyway so having one more in an observation is a moot point with the current iteration. Then having descriptive keys is not really useful anyway, because the json encoding is never meant to be human readable anyway. Picking `1` and `2` would be sufficient. So in my opinion, `b` and `u` works just the same as numbering them.

Reply:
changed to 1,2